test/app/dftb+/hybrid/periodic/GaAs-kpts-matrix-ct-np2/dftb_in.hsd

```diff
@@ -37,6 +37,8 @@ Hamiltonian = DFTB {
     0.0 0.0 0.0
   }
 
+  Solver = QR {}
```

Comment:
Probably should comment this line that the change is due to eigensolver instability with Intel xx